netlify/util/getFindingsData.ts

```diff
@@ -0,0 +1,22 @@
+import fetch from "node-fetch";
+import { AwardFinding } from "../../types/finding";
+
+/**
+ * @async
+ * @returns {Promise<[AwardFinding]>}
+ * @throws Various errors if data cannot be fetched or processed.
+ */
+const getApiFindingsData = async () => {
```

Comment:
```suggestion
const getApiFindingsData = async (): Promise<AwardFinding[]>=> {
```

Reply:
See my note at https://github.com/code-423n4/code423n4.com/pull/8739#discussion_r1156508297 — I'm not sure we can actually use TypeScript here.

If that's correct the file should be renamed to `.js` and left with only the JSDoc annotations in comments.